art_to_ftp.py

```diff
@@ -0,0 +1,60 @@
+import io
+import flask
+import logging
+from oc_cdtapi import NexusAPI
+from ftplib import FTP
+import os
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Please do not hardcode logging level.
Please inherit it from the web-sever (*gunicorn* this case).